fe/fe-core/src/main/java/com/starrocks/http/rest/RestBaseAction.java

```diff
@@ -110,7 +122,7 @@ public String getErrorRespWhenUnauthorized(AccessDeniedException accessDeniedExc
                 List<String> activatedRoles = authorizationMgr.getRoleNamesByRoleIds(context.getCurrentRoleIds());
                 List<String> inactivatedRoles =
                         authorizationMgr.getInactivatedRoleNamesByUser(userIdentity, activatedRoles);
-                return "Access denied for user " + userIdentity  + ". " +
+                return "Access denied for user " + userIdentity + ". " +
```

Comment:
You can use ErrorCode.ERR_AUTHENTICATION_FAIL

Reply:
This change is caused by code formatting, and have nothing to do with the business logic of this PR. I have rolled it back to make this PR more focused.